dashboard/db/migrate/20170426001230_add_pd_workshop_on_map_and_funded.rb

```diff
@@ -0,0 +1,6 @@
+class AddPdWorkshopOnMapAndFunded < ActiveRecord::Migration[5.0]
+  def change
+    add_column :pd_workshops, :on_map, :boolean
+    add_column :pd_workshops, :funded, :boolean
```

Comment:
They will be updated to be so in step 2, as only at that point will we have data for all of em